### ISSUE TITLE
Command Palette should feel like VS Code command palette via screen reader

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#14155]
### Windows Terminal version

1.16.2642

### Other Software

JAWS (but probably repros with other screen readers)

### Steps to reproduce

1. Open command palette
2. type "select"
3. hear "suggestions found XXX"
4. press down
5. reads "select, select all text, ctrl+shift+a"

### Expected Behavior

In step 3, it's ok to hear "suggestions found", but it should be preceded by reading out the top result (what "enter" does). In this case, it should read "select all text, ctrl+shift+a, suggestions found XXX".

Additionally, in step 5, it should not re-read the text box. It should only read "select all text, ctrl+shift+a"

User said that N of M functionality is nice, but generally should be presented at the end. Personally, he said he doesn't care too much for N of M statements so we should prioritize the other stuff, really.

### Actual Behavior

(see above)

**Comment by carlos-zamora** [comment]
If we had time, I'd add it to the 1.17 milestone. But I'll settle for 1.18. User gets really confused when there's one result available but the user doesn't know _what_ the result is.

**Comment by carlos-zamora** [comment]
Related: https://github.com/microsoft/terminal/issues/12065
